Update E3 S1 Config

Fisheiyy opened the pull request:
I have updated the Ender 3 S1 config to better suit how it would come stock on marlin and reorganized parts of the config to match the config layouts of other ender 3 variants

KevinOConnor committed:
gcode: Handle M117 and M118 commands that start with a special character

Commit 7ef7bf60 broke the special handling for M117 commands that
start with a number or special character.  Fix that support and extend
to M118 as well.

Also improve handling of commands not separated by a space (eg,
"M117HELLO").

Signed-off-by: Kevin O'Connor <kevin@koconnor.net>

Fisheiyy committed:
update to latest official klipper commits

Fisheiyy committed:
merge blackscript\klipper branch patch-1

Fisheiyy committed:
merge latest official klipper commits

Fisheiyy committed:
update E3 S1 config

in order to better suit how the E3 S1 has marlin setup so its similar to stock setup

Fisheiyy committed:
update probe offsets

Fisheiyy committed:
add E3 S1 config to printers.test

Fisheiyy committed:
fix newline build error

Fisheiyy committed:
Revert "update to latest official klipper commits"

This reverts commit 4af5571f48dc5c93748bfee1aa74be1f658e3a08.

BlackScript commented:
hm it shows conflicting so it didnt let me merge it ... not relly know how it could be changed need to read about howto github :D

BlackScript commented on klippy/gcode.py:
```diff
@@ -27,20 +27,19 @@ def get_commandline(self):
     def get_command_parameters(self):
```
thats not part of the config file ... so please remove that change from request

Fisheiyy replied:
wait when did that appear?

Fisheiyy replied:
that doesnt appear on my pr that i put for the official klipper repository just incase https://github.com/Klipper3d/klipper/pull/5287

BlackScript replied:
it shows me that you want to change 3 files ... if you go here and show your request and get on "files changed" tab 
probably this is because you not forked this repo and ask to pull and want to pull from master which probably changed sometimes other files thats not needed here ... as far as i understand it should only changed from forks so that really only that files changed which needed

Fisheiyy committed:
change safe_home xy

was watching a E3 S1 unboxing and setup on youtube and saw the z safe home position values

Fisheiyy committed:
Update printer-creality-ender3-s1-2021.cfg

https://www.reddit.com/r/3Dprinting/comments/rrpokz/ender_3_s1_first_impressions_and_pictures/ M503 dump used as reference
